js/sdk/src/v2/multiplexedWebsocket.ts

```diff
@@ -68,6 +75,13 @@ export class MultiplexedWebsocket {
           : throwError(() => err)
       }),
     )
+
+    activeRequestInternals.register(sym, {
+      serviceId: serviceId,
+      observable: request,
```

Comment:
What’s the intended use of the response observable here? As a user I’d want to have the request timestamp and the payload in here.

Reply:
I am not sure. Initially, I thought this was useful if the app wanted to close the request.
I'll add a timestamp and payload.

Reply:
My design idea of this feature is only related to observability. Making it possible to influence the behaviour of the app via this route sounds dangerous to me: cancellation and timeouts should be done by the code performing the requests, not via a global entrypoint.